cmd/kwil-admin/nodecfg/toml_test.go

```diff
@@ -74,34 +75,62 @@ func Test_IncrementingPorts(t *testing.T) {
 		input  string
 		amount int
 		want   string
+		fail   bool
 	}
 
 	testcases := []testcase{
 		{
 			input:  "localhost:26656",
 			amount: 1,
-			want:   "localhost:26657",
+			want:   "tcp://localhost:26657",
```

Comment:
Hmmm this seems undesirable for our service settings, right?  I'm surprised it's works for the http setting for example. 

"tcp://" is not a real scheme, just a strange cometbft thing. Seems unfortunate to adopt that for our http rpc service. 

Seem weird or ok?

Reply:
Yeah, agreed. Let me fix that

Reply:
Yeah, agreed. Let me fix that